types/jss/index.d.ts

```diff
@@ -96,6 +100,15 @@ export interface RuleOptions {
 	index: number;
 	className: string;
 }
+export declare class SheetsRegistry {
+    constructor();
+    registry: ReadonlyArray<StyleSheet<any>>;
+    index(): number;
```

Comment:
In TypeScript, you can represent this as `readonly index: number;`

Reply:
@kof what exactly needs to be getter?

@appsforartists I think this is correct, because it is a function which returns a number. Added a test to represent this 6b53e1fbde3df53a8c2b2feae2188e25e7e970ee . I assume this returned number is immutable, because it is a number

Reply:
@daugsbi its not a function, its a getter https://github.com/cssinjs/jss/blob/master/src/SheetsRegistry.js#L14 

```registry.index // 0```

Reply:
Thanks for your clarification. Updated the PR